javascript/javascript1/week2/FlightBookingFunction.js

```diff
@@ -0,0 +1,29 @@
+function getFullname (firstName,surName,useFormalName,gender)
+{
+    if(firstName === " " || surName=== " ")
```

Comment:
if i call your function with `getFullname("","",false,"male")` it returns "", consider how you would both check for an empty string, whitespace and null?